project-base/app/tests/App/Smoke/Http/RouteConfigCustomization.php

```diff
@@ -111,6 +111,9 @@ private function filterRoutesForTesting(RouteConfigCustomizer $routeConfigCustom
             })
             ->customizeByRouteName('admin_flag_delete', function (RouteConfig $config) {
                 $config->skipRoute('Deletion of flag from ShopSys is disabled.');
+            })
+            ->customizeByRouteName('admin_superadmin_customer_user_role_group_delete', function (RouteConfig $config) {
+                $config->skipRoute('All customer user role groups are assigned to demo customer users.');
```

Comment:
but even in that case it should return the 200 response, or?

Reply:
Thanks for pointing it out. It was a tricky one, but it is fixed now: https://github.com/shopsys/shopsys/pull/3323/commits/356d89d64951e665481113ac0be228d6d8f25821